config/webhook/manifests.yaml

```diff
@@ -59,3 +59,82 @@ webhooks:
         scope: '*'
     sideEffects: None
     timeoutSeconds: 15
+
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating.webhook.kyma-project.io
```

Comment:
since the name of the validating webhook is `validating-webhook.kyma-project.io`, can we have the name of the mutating webhook to be `mutating-webhook.kyma-project.io`? just to have a consistent pattern

Reply:
The name aligned with other webhooks